extras/rya.manual/src/site/markdown/loaddata.md

```diff
@@ -92,29 +92,56 @@ The default "format" is RDF/XML, but these formats are supported : RDFXML, NTRIP
 
 ## Bulk Loading data
 
-Bulk loading data is done through Map Reduce jobs
+Bulk loading data is done through Map Reduce jobs.
 
 ### Bulk Load RDF data
 
-This Map Reduce job will read files into memory and parse them into statements. The statements are saved into the store. Here is an example for storing in Accumulo:
+This Map Reduce job will read files into memory and parse them into statements. The statements are saved into the triplestore. 
+Here are the steps to prepare and run the job:
+
+  * Load the RDF data to HDFS. It can be single of multiple volumes and directories in them.
```

Comment:
The last sentence should be reworded.  "It can be single or multiple volumes and can have directories in them."